tests/page-objects/default/tasks/tasks.wdio.page.js

```diff
@@ -75,6 +76,19 @@ const openTaskById = async (id, taskType) => {
   await $(TASK_FORM_SELECTOR).waitForDisplayed();
 };
 
+const scrollToLastTaskItem = async () => {
+  await browser.execute(() => {
+    $('.content-row:last-child').get(0).scrollIntoView();
+  });
+};
```

Comment:
I have tried this and it doesn't apply to this context

Reply:
What do you mean? Does it fail?

Reply:
yes it doesn't perform the scroll

Reply:
I was checking why you can not use the `loadNextInfiniteScrollPage` method from [common.wdio.page.js](https://github.com/medic/cht-core/blob/master/tests/page-objects/default/common/common.wdio.page.js#L469-L474), and it was because the `.items-container` is a `div` that is present in reports and contacts, but not on tasks, but if you change it to use `.inbox-items` for example it would work for contacts, reports and tasks.

Please delete this method and use in your test the one that is in `common.wdio.page.js`, you just need to modify it to look like this:

```.js
const loadNextInfiniteScrollPage = async () => {
  await browser.execute(() => {
    $('.inbox-items .content-row:last-child').get(0).scrollIntoView();
  });
  await waitForLoaderToDisappear(await $('.left-pane'));
};
```

Look at the results, all the tests would passed
![image](https://github.com/user-attachments/assets/1a4dbb49-0c56-4108-8d34-d2443162f1bf)

Reply:
Thanks for digging deep - I have updated the method and removed the new one